.gitignore

```diff
@@ -26,4 +26,6 @@ bin/
 .vscode/
 
 ### Mac OS ###
-.DS_Store
\ No newline at end of file
+.DS_Store
+
```

Comment:
Please add: .idea/
this forder is comming from IntelliJ and must not be part of the repo